actgen/change_q.py

```diff
@@ -147,7 +149,8 @@ def direction_of_change(self, old_values, new_values, action, thresh):
         similar_actions_values = np.take(difference, similar_actions, axis=-1)
         diff_actions_values = np.take(difference, diff_actions, axis=-1)
         assert similar_actions_values.shape == (len(difference), int(num_total_actions / num_different_actions))
-        assert diff_actions_values.shape == (len(difference), int(num_total_actions / num_different_actions))
+        assert diff_actions_values.shape == \
+               (len(difference), num_total_actions - int(num_total_actions / num_different_actions))
```

Comment:
My code style preference would be the following, rather than using `\`:

```
assert (diff_actions_values.shape == 
               (len(difference), num_total_actions - int(num_total_actions / num_different_actions)))
```